elpis/engines/common/objects/interface.py

```diff
@@ -67,9 +68,18 @@ class InvalidInterfaceError(Exception):
                 # We need to keep the dir and delete the contents...
                 for root, subdirectories, files in os.walk(path):
```

Comment:
Why is the os.walk necessary? Can't you move and delete entire directories with shutil?

Calling a subprocess for every subdirectory is sure to slow it down, too.

Reply:
There are only 4 directories to delete so the spun-up subprocesses shouldn't be too slow. With regards to

> Why is the os.walk necessary? Can't you move and delete entire directories with shutil?

see the above comment:

> Tempted to use shutil.rmtree? It breaks if we have mounted /state from local filesystem into the docker container. `Error is "Device or resource busy: '/state'"` We need to keep the dir and delete the contents...

Reply:
refactored